src/builder.rs

```diff
@@ -24,6 +26,12 @@ impl Builder {
 pub trait BuilderExtManual: 'static {
     fn get_object<T: IsA<Object>>(&self, name: &str) -> Option<T>;
     fn add_from_file<T: AsRef<Path>>(&self, file_path: T) -> Result<(), Error>;
+    fn connect_signals_full<
```

Comment:
I wouldn't call it `connect_signals_full` but only `connect_signals`. We can never have the other variant :)